packages/rich-text/src/Toolbar/index.tsx

```diff
@@ -129,7 +137,8 @@ const Toolbar = ({ isDisabled }: ToolbarProps) => {
   const dropdownItemsAvailable =
     isMarkEnabled(sdk.field, MARKS.SUPERSCRIPT) ||
     isMarkEnabled(sdk.field, MARKS.SUBSCRIPT) ||
-    isMarkEnabled(sdk.field, MARKS.CODE);
+    isMarkEnabled(sdk.field, MARKS.CODE) ||
+    isMarkEnabled(sdk.field, MARKS.STRIKETHROUGH);
```

Comment:
```suggestion
  const dropdownItemsAvailable = [MARKS.SUPERSCRIPT, MARKS.SUBSCRIPT, MARKS.CODE, MARKS.STRIKETHROUGH].some(mark => isMarkEnabled(sdk.field, mark)
```
less duplication and easier to extend if there should be even more marks 😅